backend/api/v1/v1_categories/migrations/0001_initial.py

```diff
@@ -11,7 +11,7 @@ class Migration(migrations.Migration):
     dependencies = [
         ('v1_data', '0023_viewjmpcount')
     ]
-
+    file_path = "./source/config/category.json"
```

Comment:
Hi @ifirmawan, remember that modifying migrations is not allowed when it's already migrated in test / production server. Please create a new migrations to handle this

Reply:
noted, on it mas